autopkg/autopkg_tools.py

```diff
@@ -12,8 +12,8 @@
 
 
 DEBUG = False
-SLACK_WEBHOOK = os.environ.get("SLACK_WEBHOOK_TOKEN", None)
-MUNKI_REPO = os.path.join(os.environ["GITHUB_WORKSPACE"], "munki_repo")
+SLACK_WEBHOOK = os.environ.get("SLACK_WEBHOOK_TOKEN", "https://hooks.slack.com/services/CHANGE_ME")
+MUNKI_REPO = os.path.join(os.getenv("GITHUB_WORKSPACE", "/tmp/"), "munki_repo")
```

Comment:
What does getenv give us over the environ dictionary?

Reply:
Gives us a default if they didn't set the GITHUB_WORKSPACE variable